### DISCUSSION TITLE
[@types/node] childProcess.fork modulePath argument may be string or URL

### DISCUSSION THREAD

**Discussion by Courey** [DefinitelyTyped/DefinitelyTyped#69812] [Issues with a @types package] [answered]
I am trying to use `new URL(import.meta.url)` when forking a process, however I am getting a typescript error because the modulePath argument is expected to be a string. 

<img width="1262" alt="Screenshot 2024-06-14 at 3 28 07 PM" src="https://github.com/DefinitelyTyped/DefinitelyTyped/assets/6545874/ebef62ee-21f0-46ab-81d7-0edd529efe9d">

I was attempting to do it that way because the [Node documentation for fork](https://nodejs.org/api/child_process.html#child_processforkmodulepath-args-options) says that node has supported either a string or URL since v.16.14.0

<img width="685" alt="Screenshot 2024-06-14 at 3 23 37 PM" src="https://github.com/DefinitelyTyped/DefinitelyTyped/assets/6545874/35a8505c-258f-4934-8741-d6119a7dd5dd">

Would it be possible to update the [type definition](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/96b75f0205b9ea81086139857c5167886cd5da26/types/node/v16/child_process.d.ts#L1386-L1387) for fork so that modulePath accepts either a string or URL? 

I appreciate your time.

[a comment by an automated account is not shown]

**Comment by Anonymous4078** [comment]
Yes make a PR

**Comment by peterblazejewicz** [comment]
It looks like omission (legit, probably docs were incomplete in the past). The implementation works both with string or URLs:
https://github.com/nodejs/node/blob/main/lib/internal/url.js#L1484

**Comment by peterblazejewicz** [comment, marked as answer]
fix: #69819 69819
